ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -266,7 +266,7 @@ export class WorkflowsList extends BasePage<RouteComponentProps<any>, State> {
                         )}
                         <div className='argo-table-list'>
                             <div className='row argo-table-list__head'>
-                                <div className='columns workflows-list__status small-1' />
+                                <div className='columns small-1 workflows-list__status' />
                                 <div className='row small-11'>
```

Comment:
small-11 ??

Reply:
This is correct. It's completed by a `small-1` right above

Reply:
It was actually incorrect, but for a different reason. Good catch :)